src/autogenerated/java/net/minestom/server/sound/BlockSoundType.java

```diff
@@ -0,0 +1,261 @@
+package net.minestom.server.sound;
+
+import java.util.HashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * AUTOGENERATED by BlockSoundTypesGenerator
+ */
+public record BlockSoundType(float volume, float pitch, @NotNull SoundEvent breakSound,
```

Comment:
It doesn't quite fit into the scheme and I didn't see that in the first review. I would prefer to solve this with some kind of registry instead of custom code generators

Reply:
It doesn't make sense in a registry, the data isn't stored in a registry in Minecraft and isn't sent in any network buffer.